desktop/src/client/client.ts

```diff
@@ -63,7 +63,7 @@ type TChannelName = keyof TChannels
 type TClientEventListener<TChannel extends TChannelName> = (payload: TChannels[TChannel]) => void
 type TClientSettings = Pick<
   TSettings,
-  "debugFlag" | "additionalCliFlags" | "dotfilesUrl" | "additionalEnvVars"
+  "debugFlag" | "additionalCliFlags" | "dotfilesUrl" | "additionalEnvVars" | "sshKeyPath"
```

Comment:
You'll need to wire it up in the client itself, otherwise it won't have an effect

Reply:
yup yup 👍